### PR TITLE
Update PeerTube to version 4.0.0

### PR DESCRIPTION
**Pull request by BDouze**
## Problem

- Update PeerTube to version 4.0.0
- And also fix the error "uv_interface_addresses returned Unknown system error 97" that is causing peertube to fail

## Solution

- Updated peertube with latest release (https://github.com/Chocobozzz/PeerTube/releases/tag/v4.0.0)
- Removed RestrictAddressFamilies in systemd service file

## PR Status

- [ x] Code finished and ready to be reviewed/tested
- [x ] The fix/enhancement were manually tested, on ARM only (raspberry pi 4)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: conf/production.yaml [hunk 1]
```diff
@@ -420,28 +508,28 @@ instance:
...
+    '# If you would like to report a security issue\n# you may report it to:\nContact: https://github.com/Chocobozzz/PeerTube/blob/develop/SECURITY.md\nContact: mailto:'
 
 services:
   # Cards configuration to format video in Twitter
   twitter:
     username: '@yunohost' # Indicates the Twitter account for the website or platform on which the content was published
     # If true, a video player will be embedded in the Twitter feed on PeerTube video share
     # If false, we use an image link card that will redirect on your PeerTube instance
-    # Change it to "true", and then test on https://cards-dev.twitter.com/validator to see if you are whitelisted
+    # Change it to `true, and then test on https://cards-dev.twitter.com/validator to see if you are whitelisted
```
Comment (tituspijean) [inline conf/production.yaml:532]:
```suggestion
    # Change it to `true`, and then test on https://cards-dev.twitter.com/validator to see if you are whitelisted
```

**Comment by tituspijean** [comment]
!testme

**Comment by fflorent** [comment]
Hi there!

First of all, thanks for your work!

Would you mind also updating nginx accordingly to the latest changes made to the example provided by the project:
 - https://github.com/Chocobozzz/PeerTube/commit/c80e458afb63915163467fe9b3b147b3b4e4657d#diff-be9f96b9b1de67284047e610821493f9a5bec86bfcdf81a7d8d6e7904474c186
 - https://github.com/Chocobozzz/PeerTube/commit/dbe91db73e8b7992b414bdc0d3088dc73dde40b6#diff-be9f96b9b1de67284047e610821493f9a5bec86bfcdf81a7d8d6e7904474c186

Thanks in advance!

**Comment by BDouze** [comment]
> Would you mind also updating nginx accordingly to the latest changes made to the example provided by the project:

done. thanks for noticing I forgot that part

**Comment by fflorent** [comment]
Hmm, weird errors. I wonder if this relates to the upgrade, I am not sure at all:

```
 ============================================
  [Test 3/10] Installation in private mode
 ============================================

Error: In use
Running: yunohost app install --force /app_folder -a "domain=sub.domain.tld&email=admin@example.com&is_public=0&port=9000&path=/&admin=package_checker&"
1767 ERROR peertube is already installed
Error: Installation failed.

--- FAIL ---

Working time for this test: 1 minute, 8 seconds (10:05:48)
```

**Comment by fflorent** [comment]
Run locally, no issue:
```
 ============================================
 Tests summary
 ============================================


Package linter:                OK  (1 warnings, 3 possible improvements)
Install (root):                OK
Install (private):             OK
Backup/restore:                OK
Upgrade:                       OK
Upgrade (f4b43fd8):            OK
Upgrade (08bf3fce):            OK
Upgrade (f3bb0200):            OK
Upgrade (ed59a268):            OK
Upgrade (83a06ca4):            OK

Level results
=============
Level 1 (Installable in at least one scenario)    OK
Level 2 (Installable in all scenarios)            OK
Level 3 (Can be upgraded)                         OK
Level 4 (Can be backup/restored)                  OK
Level 5 (No linter errors)                        OK
Level 6 (App is in a community-operated git org)  OK
Level 7 (Pass all tests + no linter warnings)
Level 8 (Maintained and long-term good quality)
Level 9 (Flagged high-quality in app catalog)

Global level for this application: 6 (App is in a community-operated git org)
```

**Comment by fflorent** [comment]
!testme

**Comment by fflorent** [comment]
LGTM ! :slightly_smiling_face:

**Review by ericgaspar — APPROVED** [review]

**Comment by ericgaspar** [comment]
Do you have poor fonts rendering in Peertube?

Capture d’écran 2

**Comment by BDouze** [comment]
nope
Screenshot 2021-12-23 at 23 42 41

**Comment by fflorent** [comment]
No problem for me either :)

Maybe purge the webbrowser cache?

**Pull request merged**